WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/home/HomeFragment.kt

```diff
@@ -21,6 +25,15 @@ class HomeFragment : BaseFragment<FragmentHomeBinding>() {
     override fun inflateBinding(layoutInflater: LayoutInflater): FragmentHomeBinding {
        return FragmentHomeBinding.inflate(layoutInflater)
     }
+
+    override fun onStart() {
```

Comment:
any specific reason why we are setting up the FAB inside of onStart and not onCreateView?